File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.java
class Solution {
    public int solution(String[] babbling) {
        int answer = 0;
        String[] a = {"aya","ye","woo","ma"};       
        
        for(int i=0; i<babbling.length; i++){
            for(int j=0; j<a.length; j++){
               babbling[i]= babbling[i].replace(a[j]," ");
            }
                if(babbling[i].trim().length()==0){
                    answer++; 
          }
        }
        return answer;
    }
}
                   

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
    var answer = 0;
    var ong = ["aya", "ye", "woo", "ma"]
    
    answer =babbling.filter((a,i)=>a.includes(ong[0])||a.includes(ong[1])||a.includes(ong[2])||a.includes(ong[3])) 
    answer = answer.map((a,i)=>a.replaceAll(/aya|ye|woo|ma/g,1)).filter((a,i)=>a=="1"||a=="11"||a=="111"||a=="1111")
    // babbling.replaceAll("/aya|ye|woo|ma/g",1) //ong.filter((a,b,i)=>ong.push(a+ong[i]))
    
    return answer.length;
}